### PR TITLE
Add and update some Unit tests in IO module

### PR DESCRIPTION
**Pull request by eryeer**
Here we add and update some Unit tests in IO module.

### CODE REVIEW

File: neo.UnitTests/IO/Caching/UT_Cache.cs [hunk 1]
```diff
@@ -253,4 +253,4 @@ public void TestDispose()
             action.ShouldThrow<ObjectDisposedException>();
         }
     }
-}
+}
```
Comment (erikzhang) [inline neo.UnitTests/IO/Caching/UT_Cache.cs]: Can you revert this unnecessary change?

Comment (eryeer) [reply neo.UnitTests/IO/Caching/UT_Cache.cs]: yeah，Of course

**Comment by eryeer** [comment]
Done for the change

**Comment by erikzhang** [comment]
@eryeer Your master branch is not accessible for merging.

**Comment by shargon** [comment]
image

**Comment by eryeer** [comment]
what about now?
